FastSurferCNN/utils/common.py

```diff
@@ -224,46 +225,46 @@ def pipeline(
 def iterate(
     pool: Executor, func: Callable[[_Ti], _T], iterable: Iterable[_Ti]
 ) -> Iterator[Tuple[_Ti, _T]]:
-    """Iterate over iterable, yield pairs of elements and func(element).
+    """
+    Iterate over iterable, yield pairs of elements and func(element).
 
     Parameters
     ----------
     pool : Executor
-        [MISSING]
+        [MISSING].
     func : Callable[[_Ti], _T]
-        function to use
+        Function to use.
     iterable : Iterable[_Ti]
-        iterable
+        Iterable.
 
     Yields
```

Comment:
This is function has the same signature as the pipeline function above. Please change it so both use Returns or Yields.